.rubocop.yml

```diff
@@ -0,0 +1,74 @@
+Style/Documentation: 
```

Comment:
Added these configurations in the .rubocop.yml as a result of a rubocop error suggesting that this config info were added to Rubocop 
<img width="533" alt="Screen Shot 2021-05-04 at 17 16 53" src="https://user-images.githubusercontent.com/36692674/117027871-69671a00-acfd-11eb-893e-b82da1f69917.png">